pkg/cmd/pull/pull.go

```diff
@@ -18,6 +22,67 @@ import (
 	"oras.land/oras-go/v2/registry/remote"
 )
 
+func runPull(ctx context.Context, opts *pullOptions) (ocispec.Descriptor, error) {
```

Comment:
In the future, yes, shouldn't be a problem. For now I think doing it sequentially is better since it leads to the output being sequential ("pulling referenced image X... pulling image Y"). We're bottlenecked by network bandwidth anyways, so I don't know that doing it in parallel would run faster.

Reply:
I do not think we can saturate the bandwidth that easily, but we can take the iterative approach.